ui/common/src/main/res/drawable/rectangle_outline_red_radius_8dp.xml

```diff
@@ -9,5 +9,6 @@
 
   <!-- The radius makes the corners rounded -->
   <corners android:radius="8dp" />
+  <solid android:color="@color/styleguide_blue" />
```

Comment:
this view is used on the carrier billing as well, it might be better to create a new one.

Reply:
done, thanks